### PR TITLE
Scopes

### PR DESCRIPTION
**Pull request by nachosoloaga**
This PR includes slides for:

- Blocks
- Variables scopes
- Function scopes

### CODE REVIEW

File: src/slides/class1.jsx [hunk 1]
```diff
@@ -1729,5 +1729,150 @@ export default () => (
...
+        <Heading margin="0px" fontSize="50px">
+          Alcance dentro de funciones
+        </Heading>
+        <Text fontSize={30}>
+          Las variables declaradas dentro de una función no pueden ser accedidas fuera de la misma. 
+          Sin embargo, una función SI puede acceder a todas las variables declaradas dentro del alcance 
+          donde la función misma fue declarada.
+        </Text>
+        <Text fontSize={30}>
+          Ejemplo a continuación.
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1813]: Let's remove this to be consistent with the rest of the examples, they don't have any introduction

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -1729,5 +1729,150 @@ export default () => (
...
+
+          <iframe src="/src/slides/class1/examples/scopes2.html"></iframe>
+        </div>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div style={{ width: '80%'}}>
+        <Heading margin="0px" fontSize="50px">
+          Ejercicio
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1856]: Can we remove this heading? I'd like to be consistent with the rest of the snippets we will use as an "exercise", so can we put `Alcance dentro de funciones` or something like that?

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -1729,5 +1729,150 @@ export default () => (
...
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Alcance de las variables
+        </Heading>
+        <Text fontSize={30}>
+          El alcance de una variable hace referencia a la región del programa donde la misma puede utilizarse.
+          En JavaScript los distintos tipos de variables vistos (var, let y const) tienen distintos alcances.
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1769]: Can we mention the scope of a function? I mean the function itself, not the variables inside that function

File: src/slides/class1.jsx [hunk 4]
```diff
@@ -1729,5 +1729,150 @@ export default () => (
...
+        </Heading>
+        <CodePane language="js">
+          {`
+            var x = 1;
+            {
+              var x = 2;
+            }
+            console.log(x); // logs 2
+
+            let x = 1;
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1792]: Can we mention [TDZ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz)? And maybe add a that link

File: src/slides/class1.jsx (hunk 3 above)
Comment (nachosoloaga) [reply src/slides/class1.jsx:1769]: Yep we can, already added it

File: src/slides/class1.jsx (hunk 4 above)
Comment (nachosoloaga) [reply src/slides/class1.jsx:1819]: Both TDZ slide and link added 🤙

**Review by JulianPasquale — APPROVED** [review]

**Pull request merged**